testsuite/gnat2goto/tests/absolute/test.out

```diff
@@ -1,4 +1,4 @@
-[1] file absolute.adb line 7 assertion: SUCCESS
-[2] file absolute.adb line 8 assertion: SUCCESS
-[3] file absolute.adb line 10 assertion: FAILURE
+[absolute.assertion.1] line 7 assertion : SUCCESS
```

Comment:
Why the space after "assertion"?

Reply:
Good question, it doesn't make a lot of sense here but the final format is supposed to be `assertion {pretty printed condition}:`, so I included it here. I can remove it and readd it in the next commit if it bothers you though.

Reply:
Or that's where the actual assertion will live? Ok.